frontend/src/routes/Governance/common/util.tsx

```diff
@@ -286,9 +286,9 @@ export function getClustersSummaryForPolicySet(
 }
 
 export function resolveExternalStatus(policy: Policy) {
-  const knownExternalManagers = ['multicluster-operators-subscription', 'argocd-application-controller']
+  const knownExternalManagerPatterns = [/multicluster-operators-subscription/, /argocd/]
```

Comment:
Could you make `multicluster-operators-subscription` be an exact match instead?

Reply:
Sure, but if they change the name of that, I get to say I told you so